mscore/schema/xml.xsd

```diff
@@ -1,4 +1,4 @@
-<?xml version='1.0'?>
+﻿<?xml version='1.0'?>
```

Comment:
Not quite sure what(why this change is, but that's what I got when replacing the existing file with the one taken from https://github.com/w3c/musicxml/blob/gh-pages/schema/xml.xsd